frontend/src/views/auth/RegisterView/RegisterForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './RegisterForm.css';
+import validate from './RegisterValidation';
+import RegisterUseForm from './RegisterUseForm';
+
+export const RegisterForm = () => {
+  const { handleChange, handleSubmit, values, errors } = RegisterUseForm(validate);
+
+  return (
+    <form className="form" onSubmit={handleSubmit}>
+      <label htmlFor="email">
+        Email
+        <input
+          name="email"
+          type="text"
+          placeholder="Enter your email"
+          value={values.email}
+          onChange={handleChange}
+        />
+        {errors.email && <p>{errors.email}</p>}
+      </label>
+      <label htmlFor="password">
+        Password
+        <input
+          name="password"
+          type="password"
+          placeholder="Enter your password"
+          value={values.password}
+          onChange={handleChange}
+        />
+      </label>
+      <label htmlFor="password2">
+        Password
+        <input
+          name="password2"
+          type="password"
+          placeholder="Enter your password"
+          value={values.password2}
+          onChange={handleChange}
+        />
+      </label>
+      {errors.password2 && <p>{errors.password2}</p>}
```

Comment:
wyśietlanie errorów per cały formularz to słaby pomysł error powinein być przypisany do pola, dla całego formularz są errory typu taki użytkownik już istnieje zazwyczaj zwrócone przez backend